src/main/osd/osd.c

```diff
@@ -406,6 +406,9 @@ void pgResetFn_osdConfig(osdConfig_t *osdConfig)
 #ifdef USE_OSD_HD
     osdConfig->canvas_cols = OSD_HD_COLS;
     osdConfig->canvas_rows = OSD_HD_ROWS;
+
+    osdConfig->camera_fov_h = 138;
+    osdConfig->camera_fov_v = 75;
```

Comment:
You need to pull these lines out of the pre-processor define or the fov won't be able to reset to these values if you're flying analog OSDs. Just move them down to the end of the function, out of the way of `ifdef USE_OSD_HD`. Or maybe it would look even cleaner if you moved them up to `osdConfig->camera_frame_width/height` :)